Controller/Adminhtml/Reindex/Save.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * Module Algolia Algoliasearch
+ */
```

Comment:
Can you remove this comment? It's not needed as it's directly in AlgoliaSearch module directory